source/game/field/obj/ObjectWoodbox.cc

```diff
@@ -0,0 +1,63 @@
+#include "ObjectWoodbox.hh"
+
+#include "game/field/CollisionDirector.hh"
+
+namespace Field {
+
+/// @addr{0x8077E5E4}
+ObjectWoodbox::ObjectWoodbox(const System::MapdataGeoObj &params)
+    : ObjectBreakable(params) {}
+
+/// @addr{0x8077E620}
+ObjectWoodbox::ObjectWoodbox() = default;
+
+/// @addr{0x8077E678}
+void ObjectWoodbox::init() {
+    ObjectBreakable::init();
+    m_initialHeight = m_pos.y;
+    m_downwardsVelocity = 0.0f;
+}
+
+/// @addr{0x8077EBB8}
+void ObjectWoodbox::calcCollisionTransform() {
+    if (!m_collision) {
+        calcTransform();
+        EGG::Vector3f nextPos = m_pos + EGG::Vector3f(0.0f, 100.0f, 0.0f);
+        m_transform.setBase(3, nextPos);
```

Comment:
Put constexpr at top of function

```suggestion
        constexpr EGG::Vector3f SIZE = EGG::Vector3f(0.0f, 100.0f, 0.0f); // Maybe you can come up with a better name
        
        m_transform.setBase(3, m_pos + SIZE);
```